notebooks/HoustonFlood2017.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
     "# STSA Data Visualization With Python\n",
     "\n",
-    "<img width=\"700\" src=\"https://raw.githubusercontent.com/IBM/visualize-data-with-python/blob/master/doc/source/images/Harvey.png\">\n",
+    "<img width=\"700\" src=\"https://ibm.github.io/visualize-data-with-python/doc/source/images/Harvey.png\">\n",
```

Comment:
you don't need gh pages for this, you almost had it right the first time, you want to use:

https://raw.githubusercontent.com/IBM/visualize-data-with-python/master/doc/source/images/Harvey.png

just remove `blob` -- to get this URL, go to the image in the repo, right click it, select open in new tab.